src/commands/curl.js

```diff
@@ -15,16 +17,46 @@ async function loadTokens () {
   };
 }
 
+function printHelp () {
+  const apiDocUrlv2 = 'https://developers.clever-cloud.com/api/v2/';
+  const apiDocUrlv4 = 'https://developers.clever-cloud.com/api/v4/';
+
+  println(`Usage: clever curl
```

Comment:
Why not console.log?

Reply:
I thought we should use the logger for such print, but I can replace it if I misunderstood